README.md

```diff
@@ -4,21 +4,35 @@
 
 This is a generic skeleton project that can be used to quickly get a
 new [cisagov](https://github.com/cisagov) GitHub
-[Packer](https://packer.io) project started.  This skeleton project
+[Packer](https://packer.io) project started. This skeleton project
 contains [licensing information](LICENSE), as well as
 [pre-commit hooks](https://pre-commit.com) and
 [GitHub Actions](https://github.com/features/actions) configurations
 appropriate for the major languages that we use.
 
 ## Pre-requisites ##
 
-This project requires a build user to exist in AWS.  The accompanying terraform
-code will create the user with the appropriate name and permissions.  This only
-needs to be run once per project, per AWS account.  This user will also be used by
-GitHub Actions.
+This project requires a build user to exist in AWS. The accompanying Terraform
+code will create the user with the appropriate name and permissions. This only
+needs to be run once per project, per AWS account. This user will also be used
+by GitHub Actions.
+
+Before the build user can be created, the following profile must exist in
+your AWS credentials file:
+
+* `cool-terraform-backend`
+
+The easiest way to set up that profile is to use our
+[`aws-profile-sync`](https://github.com/cisagov/aws-profile-sync) utility.
+Follow the usage instructions in that repository before continuing with the
+next steps. Note that you will need to know where your team stores their
+remote profile data in order to use
+[`aws-profile-sync`](https://github.com/cisagov/aws-profile-sync).
+
+To create the build user, follow these instructions:
```

Comment:
@jsf9k Do you have the same concerns here calling this a "build user" as [you did in our ansible roles](https://github.com/cisagov/ansible-role-nessus/pull/8#pullrequestreview-397446385)?  

In these packer repos, I think "build user" is appropriate.  If we are all in agreement, then I think we should change the `terraform-test-user` directory to `terraform-build-user` and update the contents accordingly (`test` -> `build`).

Reply:
I'm fine with the packer user being called a "build user".

I agree with the rest of what you said as well.

Reply:
Should the created username also be changed? Right now it is `test-skeleton-packer`, and if we change the directory to `terraform-build-user` that seems inconsistent.

Reply:
Yes, I'm good with renaming the build user to `build-skeleton-packer`.

Reply:
These changes have been implemented in ec21791.

Reply:
You missed a spot with your renaming:  https://github.com/cisagov/skeleton-packer/blob/improvements/pull_upstream_updates/terraform-build-user/backend.tf#L8

Reply:
That was only changed from `skeleton-packer-cool` to `skeleton-packer` (https://github.com/cisagov/skeleton-packer-cool/blob/590f4fc3d5c2a0963d718f824b4b07185fe5c25a/terraform-test-user/terraform.tf#L8). Both `terraform-build-user` and `terraform-post-packer` are using `skeleton-packer` as the base for their state keys.

Reply:
The `skeleton-packer` part is fine- I was talking about changing `terraform-test-user.tfstate` to `terraform-build-user.tfstate`.

Reply:
Ahh, my misunderstanding. Fixed in 8432b1a.

Reply:
Excellent!